pkg/plugin/config.go

```diff
@@ -134,5 +141,13 @@ func (cfg *PluginConfig) ToFlags(set *cli.FlagSet) *cli.FlagSet {
 		Usage:   "Hint is for what value to put as the justification.",
 	})
 
+	f.StringVar(&cli.StringVar{
+		Name:    "jira-plugin-base-url",
+		Target:  &cfg.BaseURL,
+		EnvVar:  "JIRA_PLUGIN_BASE_URL",
+		Example: "https://verily-okta-sandbox.atlassian.net/browse/",
```

Comment:
Does this actually require the trailing slash?

Reply:
Updated the the example URL